src/__tests__/use-app/use-app.test.tsx

```diff
@@ -9,6 +9,11 @@ import CustomAppWithGIP_SSGPage from './__fixtures__/custom-app-with-gip/pages/s
 import CustomAppWithGIP_GIPPage from './__fixtures__/custom-app-with-gip/pages/gip';
 import CustomAppWithGIP_Page from './__fixtures__/custom-app-with-gip/pages/page';
 
+import CustomAppWithGIPCustomProps from './__fixtures__/custom-app-with-gip-custom-props/pages/_app';
+import CustomAppWithGIPCustomProps_GIP from './__fixtures__/custom-app-with-gip-custom-props/pages/gip';
```

Comment:
Should `CustomAppWithGIPCustomProps_GIP` be `CustomAppWithGIPCustomProps_GIPPage`?

Reply:
sure, changed